src/utils/helpers.js

```diff
@@ -0,0 +1,12 @@
+export function setButtonText(
+  btn,
+  isLoading,
+  defaultText = "Save",
+  loadingText = "Saving..."
+) {
+  if (isLoading) {
+    btn.textContent = "Loading...";
```

Comment:
Please, use `loadingText` and `defaultText` to be able to change the texts if needed


```js
export function setButtonText(
  btn,
  isLoading,
  defaultText = "Save",
  loadingText = "Saving..."
) {
  if (isLoading) {
    btn.textContent = loadingText;
  } else {
    btn.textContent = defaultText;
  }
}

```